### PR TITLE
Update minimal MAF format documentation

### LINKED ISSUE #9623: Wrong description about minimal MAF
Currently, it's not requiring the genomic locations but Hugo_Symbol, HGVSp_Short, etc. It says that it is possible to create an extended MAF by running the minimal MAF through the Genome Nexus. But GN would require the genomic location and create the protein level annotations such as Hugo_Symbol and HGVSp_Short.

https://docs.cbioportal.org/file-formats/#minimal-maf-format

image

### PR DESCRIPTION
**Pull request by rmadupuri**
Fix #9623

### CODE REVIEW

File: docs/File-Formats.md [hunk 1]
```diff
@@ -639,15 +643,15 @@ The extended MAF format recognized by the portal has:
...
+6. **Start_Position (Required)**: Start position of event.
+7. **End_Position (Required)**: End position of event.
 8. **Strand (Optional)**: We assume that the mutation is reported for the + strand.
 9. **Variant_Classification (Required)**: Translational effect of variant allele, e.g. Missense_Mutation, Silent, etc.
 10. **Variant_Type <sup>1</sup>(Optional)**: Variant Type, e.g. SNP, DNP, etc.
-11. **Reference_Allele (Optional)**: The plus strand reference allele at this position.
+11. **Reference_Allele (Required)**: The plus strand reference allele at this position.
 12. **Tumor_Seq_Allele1 (Optional)**: Primary data genotype.
-13. **Tumor_Seq_Allele2 (Optional)**: Primary data genotype.
+13. **Tumor_Seq_Allele2 (Required)**: Primary data genotype.
```
[the remark this answers is not in the document]
Comment (inodb) [reply docs/File-Formats.md:654]: So should it be `Tumor_Seq_Allele1` instead?

[the remark this answers is not in the document]
Comment (rmadupuri) [reply docs/File-Formats.md:654]: @inodb, we usually consider the `Reference_Allele` and `Tumor_Seq_Alelle2` as the reference and variant alleles. And all the MAFs from public studies are built on that. The values in `Tumor_Seq_Alelle1` are the same as the `Reference_Allele` column.

[the remark this answers is not in the document]
Comment (rmadupuri) [reply docs/File-Formats.md:654]: I think the importer looks at `Reference_Allele` and `Tumor_Seq_Alelle2` columns too - but need to double check on that.

[the remark this answers is not in the document]
Comment (inodb) [reply docs/File-Formats.md:654]: gotcha ok - thanks!

File: docs/File-Formats.md [hunk 2]
```diff
@@ -610,20 +610,24 @@ The mutation data file extends the [Mutation Annotation Format](https://docs.gdc
...
+4. **End_Position (Required)**: End position of event.
+5. **Reference_Allele (Required)**: The plus strand reference allele at this position.
+6. **Tumor_Seq_Allele2 (Required)**: Primary data genotype.
+7. **Tumor_Sample_Barcode (Required)**: This is the sample ID. Either a TCGA barcode (patient identifier will be extracted), or for non-TCGA data, a literal SAMPLE_ID as listed in the clinical data file.
 
-Next to Hugo_Symbol, it is recommended to have the Entrez gene ID:
+In addition to the above columns, it is recommended to have the read counts to calculate variant allele frequencies:
+
+8. **t_alt_count (Optional, but recommended)**: Variant allele count (tumor). 
+9. **t_ref_count (Optional, but recommended)**: Reference allele count (tumor).
```
Comment (inodb) [inline docs/File-Formats.md:625]: great that it says recommended 👍

File: docs/File-Formats.md [hunk 3]
```diff
@@ -610,20 +610,24 @@ The mutation data file extends the [Mutation Annotation Format](https://docs.gdc
...
+11. **SWISSPROT (Optional)**: (annotation column) UniProtKB/SWISS-PROT name (formerly called ID) or accession code depending on the value of the `swissprot_identifier` metadatum, e.g. O11H1_HUMAN or Q8NG94. Is not required, but not having it may result in inconsistent PDB structure matching in [mutations view](https://www.cbioportal.org/index.do?cancer_study_list=brca_tcga_pub&cancer_study_id=brca_tcga_pub&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=brca_tcga_pub_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=brca_tcga_pub_gistic&genetic_profile_ids_PROFILE_MRNA_EXPRESSION=brca_tcga_pub_mrna_median_Zscores&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&data_priority=0&case_set_id=brca_tcga_pub_complete&case_ids=&patient_case_select=sample&gene_set_choice=prostate-cancer%3A-ar-signaling-%2810-genes%29&gene_list=TP53&clinical_param_selection=null&tab_index=tab_visualize&Action=Submit#mutation_details).
```
Comment (inodb) [inline docs/File-Formats.md:630]: are these still used? I don't think so? As far as I know the protein position is annotated on the fly using genome nexus? CC @onursumer @leexgh

File: docs/File-Formats.md (hunk 2 above)
Comment (inodb) [reply docs/File-Formats.md:625]: It seems like `t_alt_count` and `t_ref_count` are listed twice? They are also listed below

**Review by inodb — CHANGES_REQUESTED** [review]
Thanks so much @rmadupuri ! This is a great improvement

Some small comments. There are a lot of columns listed still that are optional and ignored. I propose that we make it a bit more clear in the text which ones are "(optional - recommend)" and "(optional - ignored)".

It could also help to list the true minimum required comments and maybe point to an example file such as:

https://github.com/genome-nexus/genome-nexus-annotation-pipeline/blob/master/test/data/minimal_example.in.txt

And then an example that e.g. has the full TCGA spec

It could be good to work with @ozguzMete here to make sure the documentation aligns with the genome-nexus-annotation-pipeline implementation

**Comment by inodb** [comment]
Let's merge as is and follow up with other improvements later

**Review by inodb — APPROVED** [review]

**Pull request merged**